### ISSUE TITLE
Image has no score (png info and tags)

### ISSUE THREAD

**Issue by winterspringsummer** [tsngo/stable-diffusion-webui-aesthetic-image-scorer#4]
- hash of web ui : https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/737eb28faca8be2bb996ee0930ec77d1f7ebd939
- hash of aesthetic image scorer : 5ccd3004f9a9efb5b8ca2bce91fe4fbb0c8070be
- os : windows 10

I installed extension with below command,
`git clone --recurse-submodules https://github.com/tsngo/stable-diffusion-webui-aesthetic-image-scorer extensions/aesthetic-image-scorer`

- I checked that the 'sac+logos+ava1-l14-linearMSE.pth' file has been downloaded.
- I checked 'Save score as EXIF or PNG Info Chunk' and 'Save score as tag (Windows Only)'

Both Web-ui and Aesthetic-Image-Scorer are up to date, for now.
I tried with jpg and png format, but the output image does not have png info or tag info about aesthetic image score.
Am I missing something during the installation process?

**Comment by tsngo** [comment]
Yeah the EXIF or PNG Info Chunk is not working due to the above calling image saved callback twice instead of the before image saved callback which handles the EXIF or PNG Info Chunk.

**Comment by winterspringsummer** [comment]
Thank you, it works now! This extension is so great. Thansk @tsngo again for your work.
